packages/status-js/package.json

```diff
@@ -34,6 +34,7 @@
     "ethereum-cryptography": "^1.0.3",
     "js-waku": "^0.30.0",
     "long": "^5.2.0",
+    "multiformats": "^11.0.1",
```

Comment:
ℹ️ https://github.com/paulmillr/scure-base could theoretically be used, but I didn't try if just turning `base58`'s result lowercase would be sufficient. Regardless, I'm importing `varint` from `multiformats` anyway.